README.md

```diff
@@ -69,7 +69,7 @@ GOOGLE_OAUTH_CREDENTIALS came from downloaded Google oAuth credentials in json f
 
 ## Running
 
-python3 ./fetch_report.py --report "supported-report-name" --config-file "path-to-config-file"
+python3 ./fetch_report.py --report "supported-report-name" --config-file "path-to-config-file" --oauth-config-file "path-to-oauth-credentials-file"
```

Comment:
Does this change have any relationship with this PR ?

Reply:
No, but it is necessary to update it.

Reply:
could you open a new PR to do this ?

Reply:
What is the problem with?

Reply:
it's not easy track a change that is hiding in a PR without relation.

Reply:
It is an update to the README, and you can always use a blame in the git history, anywise, it is done.